tinygrad/ops_cpu.py

```diff
@@ -124,10 +126,11 @@ def backward(ctx, x):
     return np.transpose(x, np.argsort(ctx.order))
 
 def inner_slice(x, arg):
-  padding = [(max(0, -p[0]), max(0, p[1]-x.shape[i])) for i,p in enumerate(arg)]
-  x = np.pad(x, padding)
-  slicee = [(p[0] + padding[i][0], p[1] + padding[i][0]) for i,p in enumerate(arg)]
-  return x[tuple([slice(x[0], x[1], None) for x in slicee])]
+  padding, slicee = [], []
```

Comment:
Again, less readable

Reply:
i thought this would be decently faster as it turns the 2 enumerate() loops into a single one but after some testing it seems negligible, will undo this.